app/src/main/java/com/hover/stax/bounties/BountyListFragment.java

```diff
@@ -80,4 +100,9 @@ public void bountyDetail(Bounty b) {
 			})
 			.showIt();
 	}
+
+	@Override
+	public void countrySelect(String countryCode) {
```

Comment:
This doesn't need to call through to another function. filterBounties is only called from here, so just put the logic in here